[master < T2488-PL] Add release notes for Lab v2.3.1 (#584)
memgraph/docs: lab/changelog.md

sidebar_label: Changelog
---

## v2.3.1 - Nov 4, 2022

### Bug Fixes

> * Fix handling single-line commented Cypher code in query execution.
> * Fix the map toggle freeze in the graph view.
> * Fix showing multiple maps in the rich collections.
> * Fix missing node and edge information on graph schema.
> * Fix the query reordering in the rich collections.
> * Fix using a previous description when adding a new query to the rich collection.

Reviewer: I would like us to start writing user-oriented release notes/changelogs... 
The shift would be from What we did to What you can now do/or not have problems with any more
For example, as a user I have no idea what half of these fixes actually do for me... or what problems they fix from my perspective

For example: 
* Writing a single-line comment in the Cypher code no longer results in an error
* "I have no idea what this actually means"
* "as well"
* All the information about nodes and edges on the graph schema is now previewed as expected. 
* "I have no idea what happened before and what this means now for me"
* A bug that would not reset the description field when adding a new query to the query collection has been fixed. 

```suggestion
* Fix handling single-line commented Cypher code in query execution.
* Fix the map toggle freeze in the graph view.
* Fix showing multiple maps in the rich collections.
* Fix missing node and edge information on graph schema.
* Fix the query reordering in the rich collections.
* Fix using a previous description when adding a new query to the rich collection.
```

Author: Good feedback! It definitely makes sense.

+ thanks a lot for few examples, 1st, 4th and 6th lines are super well written :) 

Will update the notes.

Author: I just added an update, I am not really happy with the 5th one - I think it is still complex. What do you think?

Btw I also added the missing one (the last one).

sidebar_label: Changelog
---

## v2.3.1 - Nov 4, 2022

### Bug Fixes

* Writing a single-line comment in the Cypher code no longer results in an error.

> * Having different map tiles (e.g. "light" map tile on one map view, but "dark" map tile on another map view) for multiple graph map views in the rich collection is now fixed and enabled.

Reviewer: ```suggestion
* Having different map tiles (e.g. "light" map tile on one map view, but "dark" map tile on another map view) for multiple graph map views in the rich collection is enabled and works as expected.
```

* Graph rendering freeze when toggling the map view on/off during the graph rendering process has been fixed.
* All the information about nodes and edges on the graph schema is now previewed as expected.

> * A bug that would wrongly mix query title and description when queries are reordered in the rich query collection has been fixed.

Reviewer: ```suggestion
* A bug that would mix query title and description when queries are reordered in the rich query collection has been fixed.
```

* A bug that would not reset the description field when adding a new query to the query collection has been fixed.

> * Saving a new style now works as expected. An active style is saved, not the last applied one.

Reviewer: ```suggestion
* Saving a new style now works as expected. The active style is saved, not the last applied one.
```

> * Fixed an issue when saving a style, which now saves the current GSS code, not the last applied one.

Author: I will keep the form, but change the content a bit because it doesn't match the fix exactly.
